src/network/address.cpp

```diff
@@ -271,3 +271,17 @@ void Address::print(std::ostream *s) const
 	else
 		*s << serializeString() << ":" << m_port;
 }
+
+bool Address::isLocalhost() const {
+	if (isIPv6()) {
+		static const unsigned char localhost_bytes[] =
```

Comment:
LINT is not happy with it. `// clang-format off`

Reply:
Please don't, as we'll end up with thousands of these on/off lines, please whitelist the file instead.

Reply:
don't whitelist this pretty file.

Reply:
did you see that:

```
-bool Address::isLocalhost() const {
+bool Address::isLocalhost() const
+{
```
LINT is okay for this case, and yes it prevent a wrong codestyle here :)
The problem is with the memcmp line

Reply:
Yes changes suggested by lint are fine. Whitelisting or clang on/off is no longer needed.